apps/fz_http/lib/fz_http_web/header_helpers.ex

```diff
@@ -9,12 +9,12 @@ defmodule FzHttpWeb.HeaderHelpers do
 
   def clients, do: FzHttp.Config.fetch_env!(:fz_http, :private_clients)
 
-  def proxied?, do: not (external_trusted_proxies() == false)
+  def proxied?, do: external_trusted_proxies() != []
```

Comment:
@jamilbk the old condition never returned `false`, because it was always comparing a list to boolean: `[] == false`.

Reply:
I just noticed this too. Thanks for the fix and test!